altos-rust/altos-core/libs/heap/free_list_allocator/src/test.rs

```diff
@@ -131,9 +131,9 @@ impl TestFreeList {
 
     // Helper function to check that every block in the list satisfies some condition
     // Returns false if the condition returns false for any node
-    pub fn each_free_block_satisfies<F: Fn(&free_list::BlockHeader) -> bool>
-        (&self, condition: F) -> bool {
-
+    pub fn each_free_block_satisfies<F>(&self, condition: F) -> bool
+        where F: Fn(&free_list::BlockHeader) -> bool
+    {
```

Comment:
It is probably more clear this way. On the other hand, it seems sort of inconsistent since most of the time we have brackets, we're doing:
```
fn/if/something {
}
```
and this would be the only case where we're doing it differently. I think allowing it would also be another exception we're making to the specified style guide. I'm okay with allowing it though.